docs/02-app/01-building-your-application/10-deploying/03-multi-zones.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Multi Zones
+description: Learn how to build micro-frontends using Next.js Multi Zones to deploy multiple Next.js apps under a single domain.
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+<details open>
+  <summary>Examples</summary>
+
+- [With Zones](https://github.com/vercel/next.js/tree/canary/examples/with-zones)
+
+</details>
+
+Multi Zones are an approach to micro-frontends that separate a single large application on a domain into smaller applications that each serve a set of paths. This is useful when there are collections of pages unrelated to the other pages in the application. By moving those pages to a separate zone, you can reduce the size of each application which improves build times and removes code that is only necessary for one of the zones.
+
+For example, let's say you have the following set of pages that you would like to split up:
+
+- `/blog/*` for all blog posts
+- `/docs/*` for all documentation
+- The rest of your website
+
+With multi zones support, you can create three applications that all are served on the same domain and look the same to the user, but you can develop and deploy each of the applications independently.
+
+> **Good to know**: When a user navigates from a page in one zone to a page in another zone, they will need to unload the current page and load the resources from the new page. Pages that are frequently visited together should usually live in the same zone.
+
+## How to define a zone
+
+There are no special APIs to define a new zone. A zone is a normal Next.js application where you also configure a [basePath](/docs/app/api-reference/next-config-js/basePath) to avoid conflicts with pages and static files.
+
+```js filename="next.config.js"
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  basePath: '/blog',
+}
+```
+
+The default application that will handle all paths not sent to a more specific zone does not need a `basePath`.
+
+## How to route requests to the right zone
+
+With the Multi Zones set-up, you need to route the paths to the correct zone since they are served by different applications. You can use any HTTP proxy to do this, but typically one of the Next.js applications is chosen to be the routing application.
+
+To route to the correct zone using a Next.js application, you can use [`rewrites`](/docs/app/api-reference/next-config-js/rewrites). For each path served by a different zone, you would add a rewrite rule to send that path to the domain of the other zone. For example:
+
+```js filename="next.config.js"
+async rewrites() {
+    return [
+        {
+            source: '/blog',
+            destination: `${process.env.BLOG_DOMAIN}/blog`,
```

Comment:
This could have been a skill error on my part, but yesterday my build broke when I didn't include https://. Is it required? If so, worth adding here?

![Screenshot 2024-07-10 at 12 56 34 PM](https://github.com/vercel/next.js/assets/1616865/bb8a4809-c85c-4e4f-a956-dae7a5947000)


![Screenshot 2024-07-10 at 12 56 22 PM](https://github.com/vercel/next.js/assets/1616865/483e8b75-b68f-41a1-b559-d66ad6989d99)

Reply:
I added some text to make sure it includes `scheme` and `domain`.